gxff/gxff.go

```diff
@@ -13,6 +13,8 @@ import (
 	"google.golang.org/grpc/peer"
 )
 
+var TrustedProxies = ParseTrustedProxies()
```

Comment:
lets not do this

Reply:
As in, don't have a global variable? Can you elaborate on this?

Reply:
yep lets not have a global / exported variable